### ISSUE TITLE
Numeric column is giving an error when trying to query the column

### ISSUE THREAD

**Issue by Leylan24** [isoos/postgresql-dart#128]
When my dao processing a numeric column i get the following error:

```
type 'String' is not a subtype of type 'List' in type cast
#0 PostgresBinaryDecoder.convert (package:postgres/src/binary_codec.dart:492:38)
 Codec.decode (dart:convert/codec.dart:30:34)
 _PgResultStreamSubscription.handleMessage (package:postgres/src/v3/connection.dart:509:34)
 PgConnectionImplementation._handleMessage (package:postgres/src/v3/connection.dart:328:25)
 _RootZone.runUnaryGuarded (dart:async/zone.dart:1594:10)
_BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:339:11)
_DelayedData.perform (dart:async/stream_impl.dart:515:14)
_PendingEvents.handleNext (dart:async/stream_impl.dart:620:11)
 _PendingEvents.schedule. (dart:async/stream_impl.dart:591:7)
 _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
 _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
```

i am not sure why it is being cast to a List<int> but the _decode decodes the float value correctly to the string but obviously when it casts to List<int> it fails. I am not sure how the type gets inferred?

**Comment by salim-lachdhaf** [comment]
i have the same issue, `Numeric` type are return as `String` by the plugin.

**Comment by isoos** [comment]
@Leylan24 @salim-lachdhaf: could you please help me out if this is still an issue for you on HEAD? Or maybe give me a reproducible example? I've just added a test-case and it seems to be decoded just fine:
https://github.com/isoos/postgresql-dart/blob/master/test/decode_test.dart#L285

**Comment by Leylan24** [comment]
@isoos sorry I should of closed this issue. SimonBinder fixed this issue with the commit 7bfe932d614366eca8ffaa1eb4c813e20491ffbb
